src/nvim/eval.c

```diff
@@ -15509,6 +15451,35 @@ static void f_sort(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   do_sort_uniq(argvars, rettv, true);
 }
 
+/// "stdioopen()" function
+static void f_stdioopen(typval_T *argvars, typval_T *rettv, FunPtr fptr)
```

Comment:
Why? `sockconnect()` and `stdioopen()` does completely different things. Trying to connect to network or named pipe is not the same as "using these already open fds this way". The only thing they have in common is returning handle in the same namespace (as also `jobstart()` does).